_posts/know-your-bounds.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+author: Mauro Servienti
+title: "Know your bounds. Infinite scalability doesn't exist"
+synopsis: "There is this myth that we can design today for tomorrow's unknown requirements. Somehow we believe that we can put in place a robust architecture that will scale forever, no matter what. I'm sorry, that's impossible."
+header_image: /img/posts/know-your-bounds-header.jpg
+tags:
+- architecture
+---
+
+[You Are Not Google](https://blog.bradfieldcs.com/you-are-not-google-84912cf44afb) is one of my all-time favorites.
+
+I wholeheartedly suggest reading it.
+
+> First, Understand the Problem
+> My message isn't new, but maybe it's the version that speaks to you, or maybe UNPHAT is memorable enough for you to apply it. If not, you might try Rich Hickey's talk Hammock Driven Development, or the Polya book How to Solve It, or Hamming's course The Art of Doing Science and Engineering. What we're all imploring you to do is to think! And to actually understand the problem you are trying to solve
```

Comment:
I'm not familiar with UNPHAT. Maybe add a link. Same with the other books/courses you mention.